### PR TITLE
test: make centos7 ci  easier to understand

### PR DESCRIPTION
**Pull request by idbeta**
### What this PR does / why we need it:
test: modify apisix path for centos7 ci

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by idbeta** [comment]
@spacewander Please review if you have time.

File: utils/centos7-ci.sh [hunk 1]
```diff
@@ -30,17 +30,17 @@ install_dependencies() {
...
     wget https://github.com/etcd-io/etcd/releases/download/v3.4.0/etcd-v3.4.0-linux-amd64.tar.gz
     tar xf etcd-v3.4.0-linux-amd64.tar.gz
-    cp /etcd-v3.4.0-linux-amd64/etcdctl /usr/local/bin/
+    cp ./etcd-v3.4.0-linux-amd64/etcdctl /usr/local/bin/
     rm -rf etcd-v3.4.0-linux-amd64
 
     # install test::nginx
     yum install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl
-    cp -r /tmp/apisix ./apisix
+    # cp -r /tmp/apisix ./
```
Comment (spacewander) [inline utils/centos7-ci.sh:43]: We can remove this comment.

**Review by spacewander — APPROVED** [review]

File: utils/centos7-ci.sh (hunk 1 above)
Comment (idbeta) [reply utils/centos7-ci.sh]: my fault, fixed.

File: utils/centos7-ci.sh [hunk 2]
```diff
@@ -56,20 +55,16 @@ install_dependencies() {
     sleep 3
 
     # install dependencies
-    cd apisix
+    git clone https://github.com/iresty/test-nginx.git test-nginx
```
Comment (membphis) [inline utils/centos7-ci.sh:58]: we can change it to `api7` later

**Review by membphis — APPROVED** [review]
LGTM

**Pull request merged**